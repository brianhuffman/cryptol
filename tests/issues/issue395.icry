:t (join (zero : [1000][0]))
join (zero : [1000][0])
join (zero : [1000][0]) == []
:eval join (zero : [1000][0])
:pro (\(x:[1000][0]) -> join x == [])

:t (join (zero : [1000][0][32]))
join (zero : [1000][0][32])
join (zero : [1000][0][32]) == []
:eval join (zero : [1000][0][32])
:pro (\(x:[1000][0][32]) -> join x == [])
