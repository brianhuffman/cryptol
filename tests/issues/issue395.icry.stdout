Loading module Cryptol
(join (zero : [1000][0])) : [0]
0x0
True
0x0
Q.E.D.
(join (zero : [1000][0][32])) : [0][32]
[]
True
0x0
Q.E.D.
